playwright-e2e/tests/singular/enrollment/child-non-assenting-enrollement.spec.ts

```diff
@@ -0,0 +1,353 @@
+    await assertActivityProgress(page, 'Page 2 of 3');
+    await consentForm.next();
+
+    await assertActivityHeader(page, 'Consent Form for Minor Dependent');
+    await assertActivityProgress(page, 'Page 3 of 3');
+    await consentForm.childFirstName().fill(user.secondChild.firstName);
+    await consentForm.childLastName().fill(user.secondChild.lastName);
+    await consentForm.dateOfBirth(user.secondChild.birthDate.MM, user.secondChild.birthDate.DD, user.secondChild.birthDate.YYYY);
+    await consentForm.iHaveExplainedToMyChild().check();
+    await consentForm.toKnowSecondaryFinding().check('I want to know.');
+    await consentForm.parentGuardianSignature().fill(`${user.patient.firstName} ${user.patient.lastName}`);
+    await consentForm.parentFirstName().fill(user.patient.firstName);
+    await consentForm.parentLastName().fill(user.patient.lastName);
+    await consentForm.relationShipToSubject().check('Parent');
+    await consentForm.authorizationSignature().fill(user.patient.lastName);
+    await consentForm.agree();
+
+    // on "About My Child" page
+    const aboutMyChildPage = new AboutMyChildPage(page);
+    await aboutMyChildPage.waitForReady();
+    await assertActivityHeader(page, 'About My Child');
+    await enterMailingAddress(page, {
+      fullName: `${user.secondChild.firstName} Junior ${user.secondChild.lastName}`,
+      country: user.secondChild.country.name,
+      state: user.secondChild.state.name,
+      street: user.secondChild.streetAddress,
+      city: user.secondChild.city,
+      zipCode: user.secondChild.zip,
+      telephone: user.secondChild.phone
+    }); // Fill out address with fake data
+    await aboutMyChildPage.next();
+    // Triggered validation
+    await aboutMyChildPage.suggestedAddress().radioButton('As Entered:').check();
+    await aboutMyChildPage.next({ waitForNav: true });
+
+    // on "Parental Medical Record Release Form" page
+    const medicalRecordReleaseForm = new MedicalRecordReleaseForm(page);
+    await assertActivityHeader(page, 'Parental Medical Record Release Form');
+    await assertActivityProgress(page, 'Page 1 of 3');
+    await medicalRecordReleaseForm.unableToProvideMedicalRecords().check();
+    await medicalRecordReleaseForm.enterInformationAboutPhysician();
+    await medicalRecordReleaseForm.next();
+
+    await assertActivityHeader(page, 'Parental Medical Record Release Form');
+    await assertActivityProgress(page, 'Page 2 of 3');
+    await medicalRecordReleaseForm.next();
+
+    await assertActivityHeader(page, 'Parental Medical Record Release Form');
+    await assertActivityProgress(page, 'Page 3 of 3');
+    await medicalRecordReleaseForm.parentName().fill(`${user.patient.firstName} ${user.patient.lastName}`);
+    await medicalRecordReleaseForm.parentSignature().fill(`${user.patient.firstName} ${user.patient.lastName}`);
+    //await page.waitForResponse((resp) => resp.url().includes('/answers') && resp.status() === 200);
```

Comment:
Do we need the commented line? (see also line 125)

Reply:
done